refactor: Who, Adminwho, admin_mob_info & new permission R_SKINS

littleboobs opened the pull request:
## Что этот ПР делает
1. Много чат боксов, админ инфо, админвху, обычный вху. 
2. Админ инфо актуализирована, читает теперь антаг датум. 
3. Обычный вху теперь в несколько столбцов. 
4. Добавлен лог на чек адванседвху во время игры :) 
5. Переделан админ вху. Модераторы отделены от менторов, если никого нет в сети, то показывает просто сообщения офлайна. Разработчики с пермишеном R_DEBUG теперь отображаются с соответствующей ролью в категории Разработчики.
6. Добавлен новый пермишен R_SKINS для блокирования кнопки доступа педалям к порно скинам. Проверка пермишена на set_skin_permission и borg_skins.

Референсы и порты:
- https://github.com/ParadiseSS13/Paradise/pull/26669
- https://github.com/tgstation/tgstation/pull/53839
- https://github.com/tgstation/tgstation/pull/71170

## Почему это хорошо для игры
Красиво, рефактор. Больше никакого порно.

## Демонстрация изменений
<details><summary>Порно</summary>
<p>


**Админ список игроков**
<img width="770" height="91" alt="image" src="https://github.com/user-attachments/assets/c2c09788-a9e7-4300-81c9-0d910be750b1" />
**Список игроков для игроков**
<img width="769" height="86" alt="image" src="https://github.com/user-attachments/assets/05c59b0c-9d52-4006-b29e-f70dd5d62f29" />
**Админ инфо (?)**
<img width="765" height="215" alt="image" src="https://github.com/user-attachments/assets/534cab8c-64b1-4401-bdae-19d0e4915503" />
**Красивое**
<img width="771" height="146" alt="image" src="https://github.com/user-attachments/assets/c0ab06ca-b894-4ba7-a248-b3655e3cfc95" />
**Заслужили**
<img width="766" height="145" alt="image" src="https://github.com/user-attachments/assets/ff811829-85a2-4ab3-ab86-98cb524b64a3" />
**ЕРП в дормах**
<img width="765" height="78" alt="image" src="https://github.com/user-attachments/assets/d6a7914d-3a98-486c-8a82-742a4825b35f" />


</p>
</details>

littleboobs committed:
column

littleboobs committed:
adminmoreinfo update

littleboobs committed:
improve

littleboobs committed:
adminUwU & UwU

littleboobs committed:
sex borgs

littleboobs committed:
legacy shit

littleboobs committed:
bruhhhh

littleboobs committed:
fix

littleboobs committed:
sex borgs

littleboobs committed:
razrab

littleboobs committed:
verst

ROdenFL commented:
Не многие знают, но у нас такой по раньше чем на офах появился....
https://github.com/ss220-space/Paradise/pull/5765

littleboobs commented:
> Не многие знают, но у нас такой по раньше чем на офах появился.... #5765

Знаю. :( Просто сделал отображение в админвху на минимальный разрабский флаг. Надо систему пермишенов для разрабов, кому и как и почему, пересмотреть полностью. Но заниматься этим не мне, хотя сейчас это актуально, как никогда. Хоть на улице почти осень, но на сервере весна ☺️

ROdenFL commented:
> > Не многие знают, но у нас такой по раньше чем на офах появился.... #5765
> 
> Знаю. :( Просто сделал отображение в админвху на минимальный разрабский флаг. Надо систему пермишенов для разрабов, кому и как и почему, пересмотреть полностью. Но заниматься этим не мне, хотя сейчас это актуально, как никогда. Хоть на улице почти осень, но на сервере весна ☺️

радуйся что не тебе)

littleboobs committed:
R_VIEWRUNTIMES -->R_DEBUG

Считаю это более правильно. Подготовка к пересмотру пермишенов для реальных разрабов, а не просто смотрящих рантайм.

dageavtobusnick commented on code/game/verbs/who.dm:
```diff
@@ -1,142 +1,179 @@
+/proc/get_colored_rank(rank)
+	switch(rank)
+		if("Хост")
+			return "<font color='#1ABC9C'>[rank]</font>"
```
Фонты. Нам такое не надо.

littleboobs replied:
спан стайл с хексом работает?

dageavtobusnick commented on code/game/verbs/who.dm:
```diff
@@ -1,142 +1,179 @@
-				age = "<font color='#ff0000'><b>[age]</b></font>"
-			else if(age < 10)
-				age = "<font color='#ff8c00'><b>[age]</b></font>"
+							entry += " – <font color='black'><b>МЕРТВ</b></font>"
```
Поменяй фонты на спаны

littleboobs committed:
Ревью

littleboobs committed:
Merge remote-tracking branch 'upstream/master220' into who-&-no-sex-units

littleboobs committed:
Фикс лягушки??

littleboobs committed:
Медбот фикс

littleboobs committed:
Revert "Медбот фикс"

This reverts commit 1ae7a46239fcdf93e9b5b93f02680456418b17cb.

littleboobs committed:
Revert "Фикс лягушки??"

This reverts commit bd685fb95506e7f8a01b0298698407cb3781b736.
